src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public Ball getRandomBall() {
```

Comment:
Rewrite `getRandomBall` in one line:
`return new Ball(new ColorSupplier().getRandomColor(), 
                             new Random().nextInt(100))`